apps/web/.env.example

```
@@ -1,3 +1,8 @@
+NEXT_PUBLIC_WEB_URL=url
+
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=clerk_key
+CLERK_SECRET_KEY=key
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CLERK_SECRET_KEY key should go before the NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY key

(UnorderedKey)